docs/sources/developers/plugins/extend-the-grafana-ui-with-links-and-commands.md

```diff
@@ -0,0 +1,112 @@
+# Extend the Grafana UI with links and commands
+
+Use the Plugin extensions API with your Grafana App plugins to add links to the Grafana UI. Doing so allows you to direct users to your plugins pages from various "placements" within the Grafana application.
+
+For a plugin to successfully register links it must:
+
+- Be an App plugin.
+- Be [preloaded]({{< relref "./metadata" >}}).
+- Be installed and enabled.
+
+## Available placements within Grafana
+
+A placement is a location within the Grafana application UI where a plugin can insert links or commands. All placements within Grafana start with `grafana/`. The following placements are available:
+
+- `grafana/dashboard/panel/menu`: all panel menu dropdowns found in dashboards
```

Comment:
Maybe mention the `PluginExtensionPlacements` enum here as a source of truth? ([source](https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/pluginExtensions.ts#L5))